### ISSUE TITLE
Version 1.1.0 in Mapping to The Old Code, not The Newest Code !

### ISSUE THREAD

**Issue by turtleeeeee** [Masonry/Masonry#587]
Masonry releases 1.1.0 mapping to the old code which is lack of some important features. We hope that you could update it to the newest code of your master branch in your repo.

**Comment by ifengshumu** [comment]
use this pod command : `pod 'Masonry', :git => 'https://github.com/SnapKit/Masonry', :branch => 'master'`

**Comment by CaiChenghan** [comment]
> use this pod command : `pod 'Masonry', :git => 'https://github.com/SnapKit/Masonry', :branch => 'master'`

使用master分支，跟使用1.1.0的分支，有区别吗？两个分支切换，代码一模一样。

**Comment by turtleeeeee** [comment]
> > use this pod command : `pod 'Masonry', :git => 'https://github.com/SnapKit/Masonry', :branch => 'master'`
> 
> 使用master分支，跟使用1.1.0的分支，有区别吗？两个分支切换，代码一模一样。

你真的认真对比了吗？1.1.0和master的最新提交甚至都不是同一个啊！少了一些代码的

**Comment by ifengshumu** [comment]
Of course, I've compared that, and using the master branch is the equivalent of a manual download
